hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/upgrade/UpgradeDowngradeUtils.java

```diff
@@ -211,4 +211,11 @@ static void rollbackFailedWritesAndCompact(HoodieTable table, HoodieEngineContex
       throw new HoodieException(e);
     }
   }
+
+  static String getTimelineTimezone(HoodieTableConfig tableConfig) {
+    if (tableConfig.getTimelineTimezone() != null) {
```

Comment:
if we have already added default value (L284 in HoodieTableConfig), how come tableConfig.getTimelineTimezone could be null.

Reply:
This is to check for someone upgrading from a version when there was no default.

Reply:
but if we are executing this piece of code which is going to be in 1.0.2 release, wouldn't the default value kick in automatically from the in memory default value. 
even though hoodie.properties may not contain it for table version 6 table

Reply:
you're right.. upgrade is a step function and even when someone is upgrading from much older version they will go through each version step by step. I think this is not needed. I will remove it.